src/applications/mhv/medications/containers/LandingPage.jsx

```diff
@@ -7,7 +7,7 @@ import { medicationsUrls } from '../util/constants';
 
 const LandingPage = () => {
   const fullState = useSelector(state => state);
-  const medicationsUrl = !fullState.user.login.currentlyLoggedIn
+  const medicationsUrl = fullState.user.login.currentlyLoggedIn
```

Comment:
would you mind noting the reason for this change on the pr summary. It would be helpful so the reviewer doesn't have to assume or reach out for clarification.

Reply:
okay